internal/backup/volumesnapshotcontent_action.go

```diff
@@ -119,30 +122,104 @@ func (p *VolumeSnapshotContentBackupItemAction) Execute(item runtime.Unstructure
 		// check if VolumeBackupSnapshot CR exists for VSC
 		VSBExists, err := util.DoesVolumeSnapshotBackupExistForVSC(&snapCont, p.Log)
 		if err != nil {
-			return nil, nil, errors.WithStack(err)
+			return nil, nil, "", nil, errors.WithStack(err)
 		}
 
 		// Create VSB only if does not exist for the VSC
 		if !VSBExists {
 			vsbClient, err := util.GetVolumeSnapshotMoverClient()
+			if err != nil {
+				return nil, nil, "", nil, errors.Wrapf(err, "error getting volumesnapshotbackup client")
+			}
 
 			err = vsbClient.Create(context.Background(), &vsb)
 
 			if err != nil {
-				return nil, nil, errors.Wrapf(err, "error creating volumesnapshotbackup CR")
+				return nil, nil, "", nil, errors.Wrapf(err, "error creating volumesnapshotbackup CR")
 			}
 
 			p.Log.Infof("Created volumesnapshotbackup %s", fmt.Sprintf("%s/%s", vsb.Namespace, vsb.Name))
+
+			// Now fetch the VSB so that we get the UID from VSB metadata and return that as operationID to be used for progress monitoring
```

Comment:
Comment should reflect the name we're using Namespace/Name rather than UID -- also, since those are already known, do we need this Get?

Reply:
We still need the get call because we use generate name for VSB CR.